pennylane/templates/subroutines/qsvt.py

```diff
@@ -285,10 +285,10 @@ def __init__(self, UA, projectors, id=None):
             "projectors": projectors,
         }
 
-        ua_wires = UA.wires.toset()
-        proj_wires = set.union(*(proj.wires.toset() for proj in projectors))
+        total_wires = qml.wires.Wires(UA.wires) + qml.wires.Wires.all_wires(
```

Comment:
You might import Wires to make the code more compact:
`total_wires = Wires(UA.wires) + Wires.all_wires([proj.wires for proj in projectors])`